h/static/scripts/hypothesis.js

```diff
@@ -1,3 +1,8 @@
+if (window.annotator) {
+    console.warn("Not deploying Hypothes.is, in order to avoid double deployment.");
```

Comment:
"Your add-on prints debugging information to the Console, which is generally not allowed in production add-ons."

Let's default to quiet and only log things when we have a very strong reason to believe it will help someone.